firestore/src/include/firebase/firestore/settings.h

```diff
@@ -155,15 +155,41 @@ class Settings final {
    */
   void set_cache_size_bytes(int64_t value);
 
-#if defined(__OBJC__)
+#if defined(__OBJC__) || defined(DOXYGEN)
   /**
-   * A dispatch queue to be used to execute all completion handlers and event
-   * handlers. By default, the main queue is used.
+   * Returns a dispatch queue that Firestore will use to execute callbacks.
+   *
+   * The returned dispatch queue is used for all completion handlers and event
+   * handlers.
+   *
+   * If no dispatch queue is explictly set by calling `set_dispatch_queue()`
+   * then a dedicated "callback thread" will be used; namely, the main thread
```

Comment:
Nit: `s/thread/queue/`? "Thread" seems a little low-level here (and IIRC, libdispatch may run callbacks on different threads for the same queue).